src/Umbraco.Core/Media/IEmbedProvider.cs

```diff
@@ -2,7 +2,7 @@
 {
     public interface IEmbedProvider
     {
-        bool SupportsDimensions { get; }
+        bool SupportsDimensions { get; set; }

```

Comment:
This interface shouldn't change, there's no reason to make this setter public and part of the interface